services/cd-service/pkg/service/version_test.go

```diff
@@ -166,3 +173,167 @@ func TestVersion(t *testing.T) {
 		})
 	}
 }
+
+func TestGetManifests(t *testing.T) {
+	type testCase struct {
+		name    string
+		setup   []repository.Transformer
+		req     *api.GetManifestsRequest
+		want    *api.GetManifestsResponse
+		wantErr error
+	}
+
+	appName := "app-" + rand.String(16)
+	appNameOther := "app-other-" + rand.String(16)
```

Comment:
do we really need random values here? Why can't the apps have the same name?

Reply:
I typically do that to force myself to not use literals - you _have_ to refer to the value provided here and so make the coupling to other components of the test more explicit. I could change it though.